src/server/routes/userRoutes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { getAllUsers, createUser } from './controllers/userController.js';
+
+const router = express.Router();
```

Comment:
change name for each routr that you do to him export. 
cant be the same name for many vars